stripe/resource_stripe_plan.go

```diff
@@ -221,7 +221,7 @@ func resourceStripePlanRead(d *schema.ResourceData, m interface{}) error {
 		d.Set("interval_count", plan.IntervalCount)
 		d.Set("metadata", plan.Metadata)
 		d.Set("nickname", plan.Nickname)
-		d.Set("product", plan.Product)
+		d.Set("product", plan.Product.ID)
```

Comment:
I don't think this one is really necessary.  We pass the product's ID already, like this:

```hcl
  product  = "${stripe_product.my_product.id}"
```

I don't think there's a way to pass the object itself, as it's complaining about not receiving a string.  Maybe I'm missing something?

Reply:
Imported plan's product in tfstate was set null.
I think this is also due to type mismatch.